packages/typescript-estree/tests/ast-alignment/parse.ts

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-member-access */
```

Comment:
this is gone from master

![pichu_delet_this](https://user-images.githubusercontent.com/7462525/219226645-e93993c0-17c2-49d0-9a43-02a7033efdde.jpg)